src/vstart.sh

```diff
@@ -712,6 +712,12 @@ prepare_conf() {
         $(format_conf "${extra_conf}")
         $AUTOSCALER_OPTS
 EOF
+    if [ "$with_jaeger" -eq 1 ] ; then
+        wconf <<EOF
+        jaeger_agent_port = 6831
+        jaeger_tracing_enable = true
```

Comment:
This means that on vstart cluster that run with `--jaeger` the tracing would be enabled globally on start, when we just want it to deploy jaeger services.

I think it'll be better to keep it disabled. and if we would like to enable tracing on start up we can use the `-o jaeger_tracing_enable=true` arg

Reply:
agree with you.

> I think it'll be better to keep it disabled. 

done

@zenomri @yuvalif